require.js

```diff
@@ -69,13 +67,11 @@
     if (loaderPath == null) loaderPath = '/';
     path = unalias(name, loaderPath);
 
+    if (!has.call(modules, path)) path = aliases[path];
```

Comment:
This line will only look for the alias if the module with `path` doesn't exist. Should we allow aliases to override the module (to help in e.g. mocking) instead?

```js
if (has.call(aliases, path)) path = aliases[path];
```